oracle/packages/node/tests/functions/run.rs

```diff
@@ -0,0 +1,85 @@
+#[cfg(test)]
```

Comment:
How come this needs to be annotated while the reset of the apps in the repo do not?

Reply:
A remnant of when it used to be a unit test. Should be removed since I don't think `test` is active while compiling integration tests.

Hence these tests might not even run with this cfg guard.